public/assets/styles/main.scss

```diff
@@ -215,7 +251,25 @@ GRIDS
     grid-column: play-left / profile-right;
     grid-row: profile-top / bottom-pad;
     display: grid;
+    grid-template-rows: [top] 70.5px [middle] 30px [bottom];
+    grid-template-columns: [left] auto [right];
+
+
+    #player-comments {
```

Comment:
on my site the line separating the upper and lower divs aren't showing

Reply:
Switched to em